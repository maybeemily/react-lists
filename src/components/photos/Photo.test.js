import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';

describe('Photo Component', () => {
  it('renders a photo', () => {
    const wrapper = shallow(<Photo photo="/path"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
