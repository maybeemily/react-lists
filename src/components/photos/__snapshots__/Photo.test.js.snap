// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo Component renders a photo 1`] = `
<img
  src="/path"
/>
`;
